### DISCUSSION TITLE
Access to manifest has been blocked by CORS policy

### DISCUSSION THREAD

**Discussion by Kamysek** [vercel/community#5673] [Help] [answered]
### Summary

Hello, 

I am very new with Vercel and CORS and run into the following problem:

```
Access to manifest at 'https://website.com/site.webmanifest' from origin 'https://www.website.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

I unfortunately could not find a solution to my problem. I tried various CORS setups as shown here: https://blog.logrocket.com/using-cors-next-js-handle-cross-origin-requests/ 

Does anyone have an idea how I can resolve this problem?

### Steps to Reproduce

My current CORS configuration looks the following:
```
  async headers() {
    return [
      {
        source: "/api/(.*)",
        headers: [
          { key: "Access-Control-Allow-Credentials", value: "true" },
          { key: "Access-Control-Allow-Origin",  value: "*" },
          { key: "Access-Control-Allow-Methods", value: "GET,DELETE,PATCH,POST,PUT" },
          { key: "Access-Control-Allow-Headers", value: "X-CSRF-Token, X-Requested-With, Accept, Accept-Version, Content-Length, Content-MD5, Content-Type, Date, X-Api-Version" },
        ],
      },
    ]
  },
```

**Comment by amyegan** [comment, marked as answer]
Hey @Kamysek. You may need to [add the crossorigin="use-credentials" attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin#example_webmanifest_with_credentials) as suggested in https://stackoverflow.com/a/67942548. Please give that a try and let me know if it works for you 🙏

**Reply by Kamysek** [reply]
Hello @amyegan. I set the manifest URL when I return metadata(): Promise<Metadata> . I do not see an option how I can set the cross origin for the Metadata?

**Reply by amyegan** [reply]
I don't know how your project is set up, so I can't make any specific recommendations 😅
